### ISSUE TITLE
make the test latitude damper run in Python 3

### ISSUE THREAD

**Issue by burlen** [LBL-EESA/TECA#206]
```
pipin:~/TECA/bin$ctest -R py_ -I 5,5,1 --verbose
UpdateCTestConfiguration  from :/Users/bloring/TECA/bin/DartConfiguration.tcl
Parse Config file:/Users/bloring/TECA/bin/DartConfiguration.tcl
 Add coverage exclude regular expressions.
SetCTestConfiguration:CMakeCommand:/usr/local/Cellar/cmake/3.13.3/bin/cmake
UpdateCTestConfiguration  from :/Users/bloring/TECA/bin/DartConfiguration.tcl
Parse Config file:/Users/bloring/TECA/bin/DartConfiguration.tcl
Test project /Users/bloring/TECA/bin
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 53
    Start 53: py_test_latitude_damper

53: Test command: /usr/local/bin/python3 "/Users/bloring/TECA/test/python/test_latitude_damper.py" "361" "181" "10" "py_test_latitude_damper_%t%.%e%"
53: Test timeout computed to be: 1500
53:   File "/Users/bloring/TECA/test/python/test_latitude_damper.py", line 51
53:     mesh.set_time_step(0L)
53:                         ^
53: SyntaxError: invalid syntax
1/1 Test #53: py_test_latitude_damper ..........***Failed    0.04 sec

0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.05 sec

The following tests FAILED:
	 53 - py_test_latitude_damper (Failed)
Errors while running CTest
```

there may be other 2 to 3 issues

**Comment by burlen** [comment]
#see also #205 , #165

**Comment by elbashandy** [comment]
This was Fixed in #249
